pkg/kubelet/kubelet_node_status.go

```diff
@@ -359,24 +359,24 @@ func (kl *Kubelet) initialNode(ctx context.Context) (*v1.Node, error) {
 			node.Annotations = make(map[string]string)
 		}
 
-		klog.V(2).Infof("Setting node annotation to enable volume controller attach/detach")
+		klog.V(2).InfoS("Setting node annotation to enable volume controller attach/detach")
 		node.Annotations[volutil.ControllerManagedAttachAnnotation] = "true"
 	} else {
-		klog.V(2).Infof("Controller attach/detach is disabled for this node; Kubelet will attach and detach volumes")
+		klog.V(2).InfoS("Controller attach/detach is disabled for this node; Kubelet will attach and detach volumes")
 	}
 
 	if kl.keepTerminatedPodVolumes {
 		if node.Annotations == nil {
 			node.Annotations = make(map[string]string)
 		}
-		klog.V(2).Infof("Setting node annotation to keep pod volumes of terminated pods attached to the node")
+		klog.V(2).InfoS("Setting node annotation to keep pod volumes of terminated pods attached to the node")
 		node.Annotations[volutil.KeepTerminatedPodVolumesAnnotation] = "true"
 	}
 
 	// @question: should this be place after the call to the cloud provider? which also applies labels
 	for k, v := range kl.nodeLabels {
 		if cv, found := node.ObjectMeta.Labels[k]; found {
-			klog.Warningf("the node label %s=%s will overwrite default setting %s", k, v, cv)
+			klog.InfoS("the node label will overwrite default setting", "label", k, "label value", v, "default setting", cv)
```

Comment:
I don't think we should be using keys with spaces like "label value", maybe just "value" and "default"?

Reply:
ok,I agree with you and I fixed it.